website/styles.css

```
@@ -728,16 +792,114 @@ h1 em { font-style: normal; color: var(--accent); }
 .footer-inner a { color: var(--muted); text-decoration: none; }
 .footer-inner a:hover { color: var(--text); }
+.footer-contact-fallback { font-size: 0.8rem !important; opacity: 0.85; }
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)